static/js/chart/draw.ts

```diff
@@ -427,10 +429,174 @@ function drawLineChart(
   }
 }
 
+
+/**
+ * Draw a group of line charts with different goeId.
```

Comment:
Nit: goeId spelling

Though, can we generalize this so that it can be grouped by other measures, not just geoId?  That would be useful for the landing pages as well.

Reply:
Typo fixed.